riocli/vpn/disconnect.py

```diff
@@ -44,10 +45,23 @@ def disconnect(ctx: click.Context):
                 fg=Colors.RED)
             raise SystemExit(1)
 
+        try:
+            cleanup_hosts_file()
+        except Exception as e:
+            click.secho(f'{Symbols.WARNING} Could not clean '
+                        f'up hosts file: {str(e)}', fg=Colors.YELLOW)
+
         click.secho(
             '{} You have been disconnected from the project\'s VPN'.format(
                 Symbols.SUCCESS),
             fg=Colors.GREEN)
     except Exception as e:
         click.secho(str(e), fg=Colors.RED)
         raise SystemExit(1) from e
+
+
+def cleanup_hosts_file():
```

Comment:
suggestion: Can we move the hosts file logic to util.py? Also, let's add the cleanup_hosts_file in `project select` and `auth login` commands as well.